docker/Dockerfile.headless

```diff
@@ -0,0 +1,9 @@
+ARG PARENT_IMAGE=rlworkgroup/garage-base
+FROM $PARENT_IMAGE
+
+# Pull CodeClimate reporter
```

Comment:
perhaps you can do this in .travis.yml to avoid splitting the entrypoints? if the file is in code/garage it should get pulled into the image.